rosetta-testing/src/test/java/com/regnosys/rosetta/formatting2/FormatterServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package com.regnosys.rosetta.formatting2;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.xtext.serializer.ISerializer;
+import org.eclipse.xtext.testing.InjectWith;
+import org.eclipse.xtext.testing.extensions.InjectionExtension;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.io.Resources;
+import com.regnosys.rosetta.tests.RosettaInjectorProvider;
+
+@ExtendWith(InjectionExtension.class)
+@InjectWith(RosettaInjectorProvider.class)
+public class FormatterServiceTest {
+	@Inject CodeFormatterService formatterService;
+	@Inject Provider<ResourceSet> resourceSetProvider;
+	@Inject ISerializer serializer;
+	
+	@Test
+	void formatSingleDocument() throws IOException, URISyntaxException {
```

Comment:
Since `formatSingleDocument` and `formatMultipleDocuments` share a lot of code, I think it would improve maintainability if we added a little test utility method to this class. E.g., you pass it a list of files, and the test will format those files and compare them to their expectation.